utils/novnc_proxy

```diff
@@ -107,8 +109,15 @@ while [ "$*" ]; do
     esac
 done
 
+if [ "$LISTEN" != "$PORT" ]; then
+    HOST=${LISTEN%:*}
+    PORT=${LISTEN##*:}
+    # if no host was given, restore hostname
+    [ "$HOST" = "$PORT" ] && HOST=$(hostname)
+fi
+
 # Sanity checks
-if bash -c "exec 7<>/dev/tcp/localhost/${PORT}" &> /dev/null; then
+if bash -c "exec 7<>/dev/tcp/${HOST}/${PORT}" &> /dev/null; then
```

Comment:
Just disable the sanity check if we give the `--listen` option. Then we can remove the $HOST variable altogether.

Reply:
You will still get an error (exception):
```
OSError: [Errno 98] Address already in use
Failed to start WebSockets proxy
```